inc/error.inc.php

```diff
@@ -0,0 +1,2 @@
+<?php
+echo "<div class='error-zone'>$errors</div>"; // Displays errors in the bottom right of a view
```

Comment:
Could we call this file something more meaningful like `error-zone` or `render-errors`?

Also, seeing as it is mostly html, how do you feel about this?
```html
<div class='error-zone'><?php echo $errors ?></div>
```